Filter/ModelFilter.php

```diff
@@ -96,33 +96,17 @@ protected function handleMultiple(ProxyQueryInterface $queryBuilder, $alias, $da
         $parameterName = $this->getNewParameterName($queryBuilder);
 
         if (isset($data['type']) && $data['type'] == EqualType::TYPE_IS_NOT_EQUAL) {
-            $this->applyWhere($queryBuilder, $queryBuilder->expr()->notIn($alias, ':'.$parameterName));
-        } else {
-            $this->applyWhere($queryBuilder, $queryBuilder->expr()->in($alias, ':'.$parameterName));
-        }
+            $or = $queryBuilder->expr()->orX();
 
-        $queryBuilder->setParameter($parameterName, $data['value']);
-    }
+            $or->add($queryBuilder->expr()->notIn($alias, ':'.$parameterName));
 
-    /**
-     * @param ProxyQueryInterface|QueryBuilder $queryBuilder
-     * @param string                           $alias
-     * @param mixed                            $data
-     *
-     * @return mixed
-     */
-    protected function handleModel(ProxyQueryInterface $queryBuilder, $alias, $data)
-    {
-        if (empty($data['value'])) {
-            return;
-        }
-
-        $parameterName = $this->getNewParameterName($queryBuilder);
+            $or->add($queryBuilder->expr()->isNull(
+                sprintf('IDENTITY(%s.%s)', $queryBuilder->getRootAlias(), $this->getFieldName())
```

Comment:
`getRootAlias` is deprecated. Please use `current(($queryBuilder->getRootAliases()))` instead (notice the double parenthesis)